integration_tests/mockApis/auth.ts

```diff
@@ -2,16 +2,17 @@ import jwt from 'jsonwebtoken'
 import type { Response } from 'superagent'
 
 import tokenVerification from './tokenVerification'
+import type { ApplicationRoles } from '../../server/middleware/roleBasedAccessMiddleware'
 import { getMatchingRequests, stubFor } from '../../wiremock'
 
 const mockedUsername = () => {
   return 'USER1'
 }
 
-const createToken = () => {
+const createToken = ({ authorities = [] }: { authorities?: Array<ApplicationRoles> }) => {
```

Comment:
I'm not quite familiar with this syntax. Is it saying destructure `authorities` if passed in an object argument, otherwise set it to an empty array?

Then on line 103, destructure `authorities` from the object argument even if undefined?

Perhaps this is just stretching the limits of my TypeScript knowledge(!), but would this not work the same?

```ts
// L12
const createToken = (authorities: Array<ApplicationRoles> = []) => {

// L103
const token = (authorities?: Array<ApplicationRoles>) =>

// L115
        access_token: createToken(authorities),

// L167-169
    authorities = []

// L176
      token(authorities),
```

Reply:
Yeah pretty much, I was thinking ahead really incase we needed to stub any other values on the user for future tests.  Might be a bit premature.

Reply:
Makes sense - I'm not close enough to this work to know how likely that is!